src/sentry/snuba/tasks.py

```diff
@@ -193,6 +209,7 @@ def _create_in_snuba(subscription):
         snuba_query.aggregate,
         snuba_query.environment,
         snuba_query.event_types,
+        params={"time_window": snuba_query.time_window},
```

Comment:
Should we make `time_filter` an explicit optional parameter rather than passing it via `params`? `params` is mostly intended to be passed through to `get_filter`